godel_msgs/srv/KeyenceProcessExecution.srv

```diff
@@ -0,0 +1,18 @@
+# Joint trajectory
+trajectory_msgs/JointTrajectory trajectory_approach
+trajectory_msgs/JointTrajectory trajectory_process
+trajectory_msgs/JointTrajectory trajectory_depart
+
+# wait for execution to finish before returning
+bool wait_for_execution
+
+# Use the simulated robot or the actual robot
+bool simulate
+
+# Parameters associated with the Keyence Process
+godel_msgs/ScanPlanParameters params
+
+---
+
+# Return code
+int32 code
```

Comment:
I concur with both of you. I will log issues associated with these messages and fix them before we push back any of these changes into the main branch.